index.html

```diff
@@ -54,12 +54,12 @@ <h5 >I'm a Software Developer.</h5>
         <!-- main -->
         <section class="main">
             <div id="recent-work" class="recent-work">
-                <h3 class="title">My Recent Works</h3>
+                <h1 class="title">My Recent Works</h1>
                 <hr class="recent-work-line"/>
             </div>
             <div class="image-placeholder">
                 <div class="image-part">
-                    <img class="image" src="assets/img/imageplaceholder.png" alt="">
+                    <img class="image" src="assets/img/imageplaceholder.png" alt="recent project image">
```

Comment:
You did well giving this image an alternative but don't forget to apply it to all your `<img>` tags 📝